guides/_subsections/view-archived-recordings.md

```diff
@@ -0,0 +1,75 @@
+## [Viewing Archived Recordings](#view-archived-recordings)
+There are several ways to view <i>Archived Recordings</i>. The first method is to navigate to the Archived Recordings tabs as mentioned in the previous section. See <a href="#archive-a-recording">Archive a Recording</a>.
+
+### [All-Targets Archives View](#all-targets-archives-view)
+
+The second method is to navigate to the new <i>All-Targets</i> archived recording view within the <i>Archives</i> tab on Cryostat console sidebar.
+
+The All-Targets view is a view which gathers all of Cryostat's discovered target JVM applications into one section for ease of access. In this view, we are able to interact with any archived recordings that have been saved from a source target by opening a target's nested recordings table. 
+
+<ol>
+  <li>
+    {% capture navigate-to-all-targets-include-text %}
+    <p>
+        Click on the <i>Archives</i> tab on the sidebar, and the first tab should automatically be selected as the <i>All Targets</i> Archives view. 
+    </p>
+    {% endcapture %}
+    {% include howto_step.html
+        summary="Navigate to the All-Targets Archived Recordings view"
+        image-name="navigate-to-all-targets.png"
+        caption="The option to automatically hide all targets with zero archived recordings is on by default and can be toggled. They can also be filtered in the search bar."
+        text=navigate-to-all-targets-include-text
+    %}
+
+  </li>
+  <li>
+    {% capture click-on-source-target-include-text %}
+    <p>
+        Clicking the dropdown arrow next to a target's alias will list any archived recordings originating from that source target.
+    </p>
+    {% endcapture %}
+    {% include howto_step.html
+        summary="Select a source target application"
+        image-name="view-archives-recordings-1.png"
+        text=click-on-source-target-include-text
+    %}
+  </li>
+</ol>
+
+### [All-Archives Archives View](#all-archives-archives-view)
+
+The third method is to navigate to the new <i>All-Archives</i> archived recording view within the <i>Archives</i> tab on Cryostat console sidebar. 
+
+The All-Archives view is a view which queries Cryostat's internal file-system for any archived recordings that have been created during runtime and the directories that contain them. A Cryostat file-system directory is created and used to store archived recordings that have been saved from an active target JVM. The web-client maps the directories into nested recordings tables, laid out in rows.
```

Comment:
replace "file-system" with simply "store" or "storage" and other phrasing so that this description is less tied to implementation details, like describing how Cryostat creates a directory for each target JVM. From the end user or API client's POV these details are not actually visible nor important, really.

"during runtime" seems to imply that if I restart Cryostat, this view might not show archived recordings obtained during the previous lifecycle.